website/content/docs/reference/sentinel-policy.mdx

```diff
@@ -74,6 +74,26 @@ The following top-level objects are available to policies in the
 * `node_pool`: the node pool of the node the volume has been placed on. This is
   a [Sentinel Node Pool Object](#sentinel-node-pool-objects).
 
+### submit-csi-volume scope
+
+The following top-level objects are available to policies in the
+`submit-csi-volume` scope automatically, without an explicit import.
+
+* `volume`: the submitted volume. This is a [Sentinel CSI Host Volume
+  Object](#sentinel-csi-host-volume-objects).
+* `existing_volume`: the previous version of the volume. If `volume_exists` is
+  true, this is always non-nil. This is also a [Sentinel CSI Host
+  Volume Object](#sentinel-csi-host-volume-objects). `volume_exists`: a
+  boolean field that indicates that a previous version of the volume exists.
+* `nomad_acl_token`: the ACL token the job was submitted with. This is a
+  [Sentinel Nomad ACL Token Object](#sentinel-acl-token-objects).
+* `namespace`: the namespace the job is in. This is a [Sentinel Nomad Namespace
+  Object](#sentinel-namespace-objects).
+* `node`: the node the volume has been placed on. This is a [Sentinel Node
+  Object](#sentinel-node-objects).
+* `node_pool`: the node pool of the node the volume has been placed on. This is
+  a [Sentinel Node Pool Object](#sentinel-node-pool-objects).
```

Comment:
I'm looking at the ENT PR and realizing we don't have access to these values in the CSI create workflow, because we're not creating them in this RPC. We'll want to remove this.